data.csv

```diff
@@ -1,162 +1,3 @@
-2025‑06‑09 15:01:08,14392,5714,219792,183328,746,573,27969,17016,3778,1537
-2025‑06‑09 18:01:07,14497,5785,223178,186266,764,586,27988,17026,3794,1546
-2025‑06‑09 21:01:07,14573,5830,226196,188954,782,597,28013,17050,3823,1559
-2025‑06‑10 00:02:00,14638,5867,228545,191005,795,608,28045,17078,3844,1568
-2025‑06‑10 03:23:13,14660,5895,230792,192872,805,614,28044,17068,3867,1585
-2025‑06‑10 06:01:01,14693,5938,232680,194475,821,622,28056,17079,3883,1599
-2025‑06‑10 09:01:55,14738,5986,234733,196223,836,634,28080,17095,3895,1612
-2025‑06‑10 12:01:23,14634,6025,237435,198531,847,644,28118,17133,3901,1614
-2025‑06‑10 15:01:15,14703,6072,240364,201100,859,649,28146,17158,3907,1617
-2025‑06‑10 18:01:12,14790,6127,243661,203958,874,656,28171,17173,3910,1617
-2025‑06‑10 21:01:17,14921,6207,246577,206404,889,670,28204,17205,3918,1624
-2025‑06‑11 00:02:05,15000,6257,249119,208528,903,677,28229,17213,3926,1631
-2025‑06‑11 03:22:41,15078,6302,251304,210456,912,683,28157,17141,3931,1636
-2025‑06‑11 06:01:07,15172,6365,253021,211954,921,691,28178,17152,3935,1637
-2025‑06‑11 09:01:37,15229,6391,255138,213747,927,697,28205,17181,3938,1638
-2025‑06‑11 12:01:15,15284,6425,257837,216098,927,697,28233,17202,3939,1640
-2025‑06‑11 15:01:03,15378,6489,260737,218669,935,702,28230,17203,3940,1638
-2025‑06‑11 18:01:07,15462,6546,263509,221046,952,712,28213,17184,3971,1660
-2025‑06‑11 21:01:14,15561,6607,266480,223695,964,720,28226,17186,3991,1679
-2025‑06‑12 00:02:07,15664,6675,268861,225836,977,729,28249,17205,4028,1704
-2025‑06‑12 03:21:37,15704,6709,271237,227923,983,735,28283,17227,4031,1711
-2025‑06‑12 06:01:00,15728,6743,273103,229529,988,739,28317,17253,4031,1711
-2025‑06‑12 09:01:02,15768,6794,275293,231473,998,747,28350,17276,4048,1719
-2025‑06‑12 12:01:10,15794,6824,277604,233461,1008,753,28350,17293,4075,1730
-2025‑06‑12 15:01:03,15904,6878,280944,236477,1018,759,28378,17314,4082,1733
-2025‑06‑12 18:01:02,16024,6936,283522,238719,1025,760,28431,17341,4089,1737
-2025‑06‑12 21:01:11,16092,6973,285938,240838,1030,765,28449,17344,4095,1742
-2025‑06‑13 00:02:05,16150,7022,287960,242623,1032,772,28461,17349,4113,1756
-2025‑06‑13 03:22:50,16279,7080,289986,244417,1039,779,28468,17354,4123,1763
-2025‑06‑13 06:01:09,16352,7168,291532,245796,1040,781,28477,17361,4129,1766
-2025‑06‑13 09:01:04,16521,7321,293311,247344,1050,785,28509,17385,4067,1713
-2025‑06‑13 12:01:10,16562,7351,295753,249575,1062,795,28535,17406,4072,1719
-2025‑06‑13 15:01:14,16664,7407,298105,251566,1075,806,28563,17428,4045,1695
-2025‑06‑13 18:01:14,16721,7457,299655,252892,1093,828,28597,17448,4048,1699
+timestamp,copilot_total,copilot_ready,copilot_merged,codex_total,codex_ready,codex_merged,cursor_total,cursor_ready,cursor_merged,devin_total,devin_ready,devin_merged,codegen_total,codegen_ready,codegen_merged
+2025-01-01 12:00:00,1000,900,400,5000,4500,3800,100,90,70,500,450,300,200,180,80
+2025-01-02 12:00:00,1100,1000,450,5200,4700,3900,110,100,75,520,470,310,210,190,85
```

Comment:
We can't drop all the previous rows here.  Find a way to keep the previous data and augment it.